adapters/pom.xml

```diff
@@ -26,7 +25,8 @@
         <module>fiducia-adapter</module>
         <module>dkb-adapter</module>
         <module>commerzbank-adapter</module>
-    </modules>
+        <module>unicredit-adapter</module>
+  </modules>
```

Comment:
looks like indentation is messed up

Reply:
Hm...strange... Looks OK in my IDE:
![image](https://user-images.githubusercontent.com/24653053/63925553-f4850c80-ca52-11e9-945a-01e332fedae1.png)

Reply:
</modules> is clearly out of alignment

Reply:
Oh, I got what you are talking about. Fixed!